src/resolve/forceIgnore.ts

```diff
@@ -37,17 +28,9 @@ export class ForceIgnore {
     try {
       this.contents = readFileSync(forceIgnorePath, 'utf-8');
       // add the default ignore paths, and then parse the .forceignore file
-      // DO NOT CALL parseContents FOR THE NEW PARSER
-      // the new library handles it's own unix/windows file path separators, let it handle it
       this.parser = ignore().add(`${this.contents}\n${this.DEFAULT_IGNORE.join('\n')}`);
-      // TODO: START REMOVE AFTER GITIGNORE-PARSER DEPRECATED
-      // add the default and send to the old gitignore-parser
-      this.gitignoreParser = gitignoreParser.compile(this.parseContents(this.contents));
+      this.contents = this.contents.split('\n').join('\n');
```

Comment:
This is just a workaround, if you remove this line then [this test will fail](https://github.com/forcedotcom/source-deploy-retrieve/blob/a68cb515202761d290548cae7a397fe29e541d2c/test/client/retrieve.test.ts#L148).

When `forceIgnorePath == ''`, line 45 would throw an error when passing `this.contents` as `undefined` to `this.parseContents` and would leave `this.forceIgnoreDirectory` as `undefined`.